src/create_disk.sh

```diff
@@ -479,6 +479,18 @@ install_grub_cfg() {
     fi
 }
 
+# For some commands, we need to make sure to use the binary and userspace of the
+# target system. XXX: Switch to bwrap.
+chroot_run() {
+    for mnt in dev proc sys run var tmp; do
+        mount --rbind "/$mnt" "${deploy_root}/$mnt"
+    done
+    chroot "${deploy_root}" "$@"
+    for mnt in dev proc sys run var tmp; do
+        umount "${deploy_root}/$mnt"
```

Comment:
Since we used `rbind`, I think we want `umount -R`.

Reply:
or `|| :` , otherwise it fails with `target is busy`
fixed that with other parts in https://github.com/coreos/coreos-assembler/pull/3039